poppy/zernike.py

```diff
@@ -618,6 +619,12 @@ def hexike_basis_wss(nterms=9, npix=512, rho=None, theta=None,
     This function has an attributed hexike_basis_wss.label_strings for
     convenient use in plot labeling.
 
+    Historical note on ordering: The reordering here relative to the more typical
+    ordering of Zernikes is motivated by moving spherical to position 9, and using
+    just the first 8 for JWST. This is because focus and spherical are not orthogonal
+    when in hexike versions on the JWST aperture, so "spherical was booted out of the picture"
+
+
 
     Parameters
     ----------
```

Comment:
Further down on line 831 (the class docstring for `Segment_PTT_Basis`), the line should be changed from "the opd_from_zernikes or opd_expand_segments functions." to "the compose_opd_from_basis or decompose_opd_segments functions."